### ISSUE TITLE
Typescript types for GIF support

### ISSUE THREAD

**Issue by deanmoses** [mattiasw/ExifReader#266]
### Description

The newly added GIF support (thanks!) isn't represented in the TypeScript bindings.

**Additional details**
- ExifReader version: 4.19.0
- Node version: 20.6.1

**How to reproduce**
Using TypeScript in Visual Studio Code:
```
import ExifReader from 'exifreader';
const tags: ExifReader.ExpandedTags = ExifReader.load(fileContents, { expanded: true });
tags.gif; // <- Causes TypeScript error: Property 'gif' does not exist on type 'ExpandedTags'.ts(2339)
```

**Comment by mattiasw** [comment]
Of course... 😞 That should be on a pre-merge checklist for me, always forget. 😅 Will fix as soon as I can. Thanks for pointing it out.

**Comment by mattiasw** [comment]
Released as version 4.19.1.
